tests/coreos-install/util/util.go

```diff
@@ -165,6 +165,45 @@ func DownloadFile(t *testing.T, tmpDir, name string) error {
 	return nil
 }
 
+// used to create files in /run/systemd/network on the host machine if no
+// files are present for use in network unit testing
+func CreateNetworkUnits(t *testing.T) string {
+	var ret string
+	if _, err := os.Stat("/run/systemd/network"); os.IsNotExist(err) {
+		err = os.MkdirAll("/run/systemd/network", 0777)
+		if err != nil {
+			t.Fatalf("creating /run/systemd/network: %v", err)
+		}
+		ret = "/run/systemd/network"
+	} else {
+		files, err := ioutil.ReadDir("/run/systemd/network")
+		if err == nil {
+			for _, file := range files {
+				if _, err = os.Stat(file.Name()); !os.IsNotExist(err) {
```

Comment:
If `ReadDir` returns something, we don't need to stat it to make extra-sure it exists.  It should be OK to return `""` if `len(files) > 0`.